src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import Page from './Page';
```

Comment:
```suggestion
import React, { useState } from 'react';

import Page from './Page';
```
한 줄 띄워주세요~ 모듈 간에 역할이 다르기 때문에 분리해두면 좀더 보기 편할 것 같아요 :)